docs/guide/cookbook/module.md

```diff
@@ -419,7 +419,50 @@ Routes to be added must be _Array_ type even if it only has one element as you c
 
 ![route_liked_borderline](../images/route_liked.png)
 
-### 2-4. Recipe D (Use hooks)
+### 2-4. Recipe D (Create hooks)
+As you know we have two main kinds of hooks `listener` and `mutator`. In this section, we look into how to create our own ones.
```

Comment:
It suggest we have two types and we will create a third one. However, it is about creating hooks of both kinds. We have to rephrase it a bit